application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml

```diff
@@ -142,7 +144,7 @@
           ## Table header row.
           &lt;thead&gt;
             &lt;tr&gt;
-              &lt;th&gt; $services.localization.render('contrib.xpoll.user') &lt;/th&gt;
+              &lt;th&gt; $services.localization.render('contrib.xpoll.user', $usersCount) &lt;/th&gt;
```

Comment:
I'm curious: why include the total as part of the translation and not something that is displayed after for example?

For example:

```
&lt;th&gt; $services.localization.render('contrib.xpoll.user') ($usersCount) &lt;/th&gt;
```

Or styled differently with CSS

?

In this way you wouldn't:
* break the existing translation key
* allow it to work for any translation
* the count doesn't really seem to be a translation, what is there to translate?

Thx

Reply:
Re the CSS styling I was thinking about displaying the count in a different style, after the text. You wouldn't be able to do that in a translation string.

Reply:
Indeed, there is no need of breaking the translation key if what is introduced it's a number. Thanks!

Regarding de css, for me the number is easy to be seen. What are you thinking for highlighting it? 
@evalica WDYT?